app/Models/Room.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\BelongsToMany;
+
+class Room extends Model
+{
+    use HasFactory;
+
+    public $hidden = ['created_at', 'updated_at', 'pivot'];
+
+    public function type(): BelongsTo
+    {
+        return $this->belongsTo(Type::class);
+    }
+
+    public function bookings(): BelongsToMany
+    {
+        return $this->belongsToMany(Booking::class);
+    }
```

Comment:
This is an interesting one.

Based on the requirements and the front-end, rooms and bookings only needs to be a one-to-many relationship. Having said that, it's certainly possible that requirements may change in the future and we need to start allowing people to book multiple rooms. 

I think I'm happy for you to leave it as many-to-many